comp/core/secrets/secretsimpl/refresh_audit.go

```diff
@@ -0,0 +1,102 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package secretsimpl is the implementation for the secrets component
+package secretsimpl
+
+import (
+	"encoding/json"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/scrubber"
+)
+
+var (
+	cutoffLimit    = time.Hour * 24 * 90 // prune entries older than 90 days
+	timeNowFunc    = time.Now
+	hexStringRegex = regexp.MustCompile(`\b[a-fA-F0-9]+\b`)
+)
+
+// RefreshAuditRow represents when a single secret is refreshed at a moment in time
+type RefreshAuditRow struct {
+	When   time.Time
+	Handle string
+	Value  string
+}
```

Comment:
Do we need this struct to be public? It look like we can get away for no being private 😄